docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     image: tomsquest/docker-radicale
     container_name: radicale
     ports:
-      - 127.0.0.1:5232:5232
+      - 5232:5232
```

Comment:
Binding to 127.0.01 is preferable for security. The default is opening the port to all (0.0.0.0.0)